scanner/services/converters/converters_test.go

```diff
@@ -670,3 +670,51 @@ func Test_convertToNormalizedSeverity(t *testing.T) {
 	// Test nothing was added without us knowing.
 	assert.Equal(t, int(claircore.Critical), 5)
 }
+
+func Test_getVulnName(t *testing.T) {
+	testcases := map[string]struct {
+		name     string
+		links    string
+		expected string
+	}{
+		"Alpine": {
+			name:     "CVE-2018-16840",
+			expected: "CVE-2018-16840",
+		},
+		"Amazon Linux": {
+			name:     "ALAS-2022-1654",
+			expected: "ALAS-2022-1654",
+		},
+		"Debian": {
+			name:     "DSA-4591-1 cyrus-sasl2",
+			expected: "DSA-4591-1",
+		},
+		"pyup.io": {
```

Comment:
does this still happen? thought pyup.io was gone?

Reply:
I don't know if they do. Let me know.

Reply:
Let's remove this, as pyup.io is gone. Instead, it'll be common we get GHSA from the OSV source. Can we test that?

Reply:
Good.

Reply:
If GHSA have links to CVE, CVE should win AFAIK.